### PR TITLE
Desgin Screens 

### PR DESCRIPTION
**Pull request by TarekIdrees**
Complete first five screens

### CODE REVIEW

File: app/src/main/res/drawable/image_loading.png
Comment (MennaAkram) [inline app/src/main/res/drawable/image_loading.png:1]: it should be a lottie file not image

File: app/src/main/ic_launcher-playstore.png
Comment (AzizaHelmy) [inline app/src/main/ic_launcher-playstore.png:1]: why this image??
if you added it 4 splash, we will not use this way

File: app/src/main/java/com/honeycake/tictactoe/ui/composable/GameTitle.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.honeycake.tictactoe.ui.composable
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.text.style.TextAlign
+import com.honeycake.tictactoe.ui.theme.TextColor
+import com.honeycake.tictactoe.ui.theme.Typography
+
+@Composable
+fun GameTitle(){
```
Comment (AzizaHelmy) [inline app/src/main/java/com/honeycake/tictactoe/ui/composable/GameTitle.kt:10]: composable function just to call composable function without any changes !, i think no need to seperate it ?!

File: app/src/main/java/com/honeycake/tictactoe/ui/composable/spacing/padding/Spacer12.kt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.honeycake.tictactoe.ui.composable.spacing.padding
+
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
```
Comment (AzizaHelmy) [inline app/src/main/java/com/honeycake/tictactoe/ui/composable/spacing/padding/Spacer12.kt]: no need to it

**Review by AzizaHelmy — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/honeycake/tictactoe/ui/composable/GameTitle.kt (hunk 1 above)
Comment (TarekIdrees) [reply app/src/main/java/com/honeycake/tictactoe/ui/composable/GameTitle.kt:10]: I used it for each screen, So it somehow reduces the repeat of Text() composable for each screen I can just call GameTitle()

**Review by AzizaHelmy — APPROVED** [review]

**Review by MennaAkram — APPROVED** [review]

**Pull request merged**